joomla-extensions/weblinks
# [com_weblinks] Make ACL core.edit.own work (PR for 11466)

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/11466 (com_weblinks part).

#### Summary of Changes

The ACL core.edit.own is not working in com_weblinks.
This PR makes it work.

#### Testing Instructions

1. Use latest staging
2. Besides the Super User, create a user "test" added to "Administrator" group
3. Go to com_weblinks Options, Permissions tab and diable "Edit" for "Administrator" group
4. Now create a new weblink with the "test" user (use another browser or a private window)
5. Try to edit that item. You can't edit our own. **Bug**
6. Now do the same test but with a weblink category Permission. You can't edit our own. **Bug**
7. Apply patch, repeat step 5. and 6. and now you can  edit our own items.
8. Code review

Also use the two users to do a general test with the com_weblinks edit permissions to confirm all is fine.

#### Documentation Changes Required

None.

#### Notes

This problem was discovered in GsoC 2016 Multilingual project.

This is very similiar with https://github.com/joomla/joomla-cms/pull/11502 (com_newsfeeds) and https://github.com/joomla/joomla-cms/pull/11503 (com_contact).
